src/v0/documents.ts

```diff
@@ -0,0 +1,120 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import { ThBaseHandler } from '../base'
+
+export interface DocumentsOptions {
+  user?: string
+  base?: string
+}
+
+export interface DocumentsMetaQuery {
+  documentNumber?: string
```

Comment:
Why `Meta` Query?
`meta` endpoint always consumes the same filter as the regular `getAll` request.

And why `documentNumber`?
<img width="371" alt="image" src="https://github.com/tillhub/tillhub-sdk-javascript/assets/8308691/9c7fb852-42ef-4e32-bb65-a8a9ceaa5052">

Reply:
Make filters as `Record<string, unknown>` for now. I don't have an agreement about document number yet